apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```diff
@@ -303,21 +330,45 @@ const ArticleModal: React.FC<ArticleModalProps> = ({
                                 actor={actor}
                                 commentCount={object.replyCount ?? 0}
                                 last={true}
-                                layout={activityThreadParents.length > 0 ? 'modal' : 'modal'}
+                                layout={'modal'}
                                 object={object}
+                                showHeader={(canNavigateBack || (activityThreadParents.length > 0)) ? true : false}
                                 type='Note'
                                 onCommentClick={() => {
-                                    setReplyBoxFocused(true);
+                                    repliesRef.current?.scrollIntoView({
+                                        behavior: 'smooth',
+                                        block: 'center'
+                                    });
                                 }}
                             />
                         )}
                         {object.type === 'Article' && (
-                            <ArticleBody
-                                excerpt={object?.preview?.content}
-                                heading={object.name}
-                                html={object.content}
-                                image={object?.image}
-                            />
+                            <>
+                                <ArticleBody
+                                    excerpt={object?.preview?.content}
+                                    heading={object.name}
+                                    html={object.content}
+                                    image={object?.image}
+                                />
+                            </>
+                        )}
+
+                        {object.type === 'Article' && (
```

Comment:
There is already a block of for `object.type === 'Article'`. Are we not able to move this into that?

Reply:
Good catch, changing it now